electron/execution.ts

```diff
@@ -136,13 +142,15 @@ function addActionsToStepResult(steps, event) {
     ...event,
     data: {
       ...event.data,
-      actionTitles: step.actions.map((action, index) => action?.title ?? `Action ${index + 1}`),
+      actionTitles: step.actions.map(
+        (action: ActionInContext, index: number) => action?.title ?? `Action ${index + 1}`
+      ),
     },
   };
 }
 
-async function onTest(data, browserWindow) {
-  const parseOrSkip = chunk => {
+async function onTest(data: RunJourneyOptions, browserWindow: BrowserWindow) {
+  const parseOrSkip = (chunk: string) => {
```

Comment:
```suggestion
  const parseOrSkip = (chunk: string): Array<Record<string, unknown>> => {
```

I think we can annotate the return type as this without any bad result.